### ISSUE TITLE
Disable react-ga after initialization (GDPR)

### ISSUE THREAD

**Issue by HcwXd** [react-ga/react-ga#381]
Due to GDPR, I have to provide a way for users to withdraw their cookie consent. So I'm wondering is there a way to disable react-ga after initializing it?

The solution mentioned in #271 is to postpone calling initialize(). But here I'd like to disable it after calling initialize(). Now I have a workaround to set a flag and check that flag every time before sending a GA event.

**Comment by romiem** [comment]
I too am wondering about this. My plan is to try manually expiring the cookies using the 'universal-cookie' library.

```
import Cookies from 'universal-cookie';
Cookies.remove('_ga');
Cookies.remove('_gat');
Cookies.remove('_gid');
```

**Comment by PierrickI3** [comment]
@romiem Did you end up implementing your solution using `universal-cookie`? Or did you have extra tasks to perform?

**Comment by romiem** [comment]
> @romiem Did you end up implementing your solution using `universal-cookie`? Or did you have extra tasks to perform?

Yes I did end up using universal-cookie (don’t forget to set the path and domain options when calling the remove method).

I’m not sure if it is perfect solution, as I assume react-ga has still loaded the google analytics script into the page and who knows what it is doing under the hood. That said, this solution feels like it is better than doing nothing.

**Comment by wonskarol** [comment]
this might help: https://developers.google.com/analytics/devguides/collection/gtagjs/user-opt-out
